plugins/coverbrowser.koplugin/bookinfomanager.lua

```diff
@@ -114,6 +114,11 @@ local BOOKINFO_SELECT_SQL = "SELECT " .. table.concat(BOOKINFO_COLS_SET, ",") ..
                             "WHERE directory=? AND filename=? AND in_progress=0;"
 local BOOKINFO_IN_PROGRESS_SQL = "SELECT in_progress, filename, unsupported FROM bookinfo WHERE directory=? AND filename=?;"
 
+-- We need these _ litterals for them to be made available to translators.
+local UNSUPPORTED_REASONS = {
+    not_readable_by_engine = { string = "not readable by engine", translation = _("not readable by engine") },
```

Comment:
Possibly better to split up in slightly more lines with an eye on future diffs? (Just thinking aloud, not necessarily a suggestion.)

Reply:
I am allowed uneven indentation, so they would look align and easier  to catch any typo/differences ?
```lua
local UNSUPPORTED_REASONS = {
    not_readable_by_engine = {
               string = "not readable by engine",
        translation = _("not readable by engine")
    },
    too_many_interruptions_or_crashes = {
               string = "too many interruptions or crashes",
        translation = _("too many interruptions or crashes")
    }
}
```

Reply:
Or other stupider variations :)
```lua
local UNSUPPORTED_REASONS = {
                         not_readable_by_engine = {
               string = "not readable by engine",
        translation = _("not readable by engine")
    },
                         too_many_interruptions_or_crashes = {
               string = "too many interruptions or crashes",
        translation = _("too many interruptions or crashes")
    }
}
```
or
```lua
local UNSUPPORTED_REASONS = {
                     not_readable_by_engine = {
           string = "not readable by engine",
    translation = _("not readable by engine")
    },
                     too_many_interruptions_or_crashes = {
           string = "too many interruptions or crashes",
    translation = _("too many interruptions or crashes")
    }
}
```

Reply:
Fine by me, but like I said I'm not objecting to the one-line thing, just voicing a stray thought. :-)